docs/install/thetagpu.rst

```diff
@@ -8,27 +8,38 @@ ThetaGPU (Argonne LCF)
 Already installed module
 ========================
 
-This installation procedure shows you how to access the installed DeepHyper module on ThetaGPU. After logging in Theta, connect to a ThetaGPU service node:
+This installation procedure shows you how to access the installed DeepHyper module on ThetaGPU. It may be useful to wrap these commands in this ``activate-dhenv.sh`` script :
 
-.. code-block:: console
+.. code-block:: bash
+    :caption: **file**: ``activate-dhenv.sh``
 
-    $ ssh thetagpusn1
+    #!/bin/bash
 
-Then, to access Deephyper run the following commands:
+    . /etc/profile
 
-.. code-block:: console
+    module load conda/2021-09-22
+    conda activate base
+
+To then effectively call this activation script in your scripts, you can use ``source ...``, here is an exemple to test the good activation of the conda environment (replace the ``$PROJECT_NAME`` with your project, e-g: ``#COBALT -A datascience``) :
+
+.. code-block:: bash
+    :caption: **file**: ``job-test-activation.sh``
+
+    #!/bin/bash
+    #COBALT -q single-gpu
+    #COBALT -n 1
+    #COBALT -t 20
+    #COBALT -A $PROJECT_NAME
+    #COBALT --attrs filesystems=home,theta-fs0,grand,eagle
 
-    $ module load conda/2021-09-22
-    $ conda activate base
+    source activate-dhenv.sh
+    python -c "import deephyper; print(f'DeepHyper version: {deephyper.__version__}')"
 
-Finally, to verify the installation do:
+You should obtain a ``DeepHyper version: 0.4.0`` in the output cobaltlog file from this job after submitting it with :
```

Comment:
don't say which version exactly, otherwise we need to update this at each release which can be quite cumbersome